EditorExtensions/Misc/MenuItems/BundleFiles.cs

```diff
@@ -64,8 +64,8 @@ private void BeforeQueryStatus(object sender, params string[] extensions)
 
             _files = ProjectHelpers.GetSelectedFilePaths();
 
-            menuCommand.Enabled = _files.All(file => extensions.Contains(Path.GetExtension(file))) &&
-                                  _files.Count() > 1;
+            menuCommand.Enabled = _files.Any(file => extensions.Contains(Path.GetExtension(file))) &&
+                          _files.Count(file => extensions.Contains(Path.GetExtension(file))) > 1;
```

Comment:
I kept the original logic requiring that at least 2 files be required to create a bundle, but I have no problem with dropping this requirement so that single-file bundles can be created from the menu.

This might actually be useful in the case the you intend to start referencing the bundle immediately but add more files later.

Do I need to create a new pull request for this?